tests/integration/cypress.spec.js

```diff
@@ -63,23 +61,4 @@ describe('runs tests on cloud', function () {
     expect(assets['video.mp4']).toBe('video.mp4');
     expect(assets.video).toBe('video.mp4');
   });
-
-  test('job has name/tags correctly set', async function () {
```

Comment:
Why is this test removed?

Reply:
added another way to test the build and tags